mta/entities/analysis_results.py

```diff
@@ -140,3 +160,17 @@ class AnalysisResultsPage(MTANavigateStep):
 
     def step(self):
         self.prerequisite_view.navigation.select("Analysis results")
+
+
+@ViaWebUI.register_destination_for(AnalysisResults)
+class AnalysisDetailsPage(MTANavigateStep):
+    prerequisite = NavigateToSibling("AnalysisResultsPage")
+    VIEW = AnalysisDetailsView
+
+    def step(self):
+        """This will navigate to details page of latest analysis"""
+        # TODO(ghubale): Improve it to navigate to specific analysis details page
+        self.prerequisite_view.table.wait_displayed()
+        for row in self.prerequisite_view.table:
```

Comment:
This will click on all rows , we can create AnalysisDetailsView as parametrized view which will take the row number and click on that particular row .There is already "analysis_row"  view in the same class . Try to use that , you can do analysis_row.details.click.

Reply:
We need to discuss this issue. I am not getting it.

Reply:
@sshveta Thanks for this suggestion.
```
@ViaWebUI.register_destination_for(AnalysisResults)
class AnalysisDetailsPage(MTANavigateStep):
    prerequisite = NavigateToSibling("AnalysisResultsPage")
    VIEW = AnalysisDetailsView

    def step(self):
        """This will navigate to details page of latest analysis"""
        self.prerequisite_view.analysis_row(row=1).analysis_number.click()```